tests/text/test_pipeline_explore.py

```diff
@@ -1,38 +1,36 @@
-from elasticsearch import Elasticsearch
-
-from biome.text import explore
-from biome.text.data import DataSource
 import pytest
-import pandas as pd
+from biome.text import Dataset
+from biome.text import explore
 from biome.text import Pipeline
+from elasticsearch import Elasticsearch
 
 
 @pytest.fixture
-def data_source_without_mapping(tmp_path) -> DataSource:
-    data_file = tmp_path / "train.json"
-    df = pd.DataFrame({"text": ["This is a simple test"], "label": ["a"]})
-    df.to_json(data_file, lines=True, orient="records")
+def dataset_without_mapping() -> Dataset:
```

Comment:
Since `Dataset`s do not have mappings, simply call this fixture `dataset`
```suggestion
def dataset() -> Dataset:
```